incubator/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     release: {{ .Release.Name }}
   name: {{ template "elasticsearch.master.fullname" . }}
 spec:
+  serviceName: {{ template "master.fullname" . }}
```

Comment:
"master.fullname" template si missing. Should it be "elasticsearch.master.fullname"